pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --ignore=e2e_tests
```

Comment:
So the intention here is that it'll only be run via Github Actions (or if you want to manually test)?

Reply:
Exactly, yeah

Reply:
You can run `pytest e2e_tests` and it will run them, but still run `pytest` on your local machine